docs/runbooks/CDINoDefaultStorageClass.md

```diff
@@ -2,48 +2,71 @@
 
 ## Meaning
 
-This alert fires when there is no default (Kubernetes or virtualization) storage
+This alert fires when there is no default (Kubernetes or KubeVirt) storage
 class, and a data volume is pending for one.
 
-A default virtualization storage class has precedence over a default Kubernetes
+A default KubeVirt storage class has precedence over a default Kubernetes
 storage class for creating a VirtualMachine disk image.
 
 ## Impact
 
-If there is no default (k8s or virt) storage class, a data volume that requests
-a default storage class (storage class not explicitly specified) will be pending
-for one.
+If there is no default (Kubernetes or KubeVirt) storage class, a data volume
+that requests a default storage class (storage class not explicitly specified)
+will be pending for one.
 
 ## Diagnosis
 
-Get the default Kubernetes storage class:
+1. Check for a default Kubernetes storage class by running the following
+command:
+
 ```bash
 $ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
 ```
 
-Get the default virtualization storage class:
+2. Check for a default KubeVirt storage class by running the following command:
+
 ```bash
 $ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
 ```
 
-To set the default Kubernetes storage class if needed:
+3. To set the default Kubernetes storage class if needed:
+
 ```bash
 $ kubectl patch storageclass <storage-class-name> -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
 ```
 
-To set the default virtualization storage class if needed:
+4. To set the default KubeVirt storage class if needed:
+
 ```bash
 $ kubectl patch storageclass <storage-class-name> -p '{"metadata": {"annotations":{"storageclass.kubevirt.io/is-default-virt-class":"true"}}}'
 ```
```

Comment:
I would remove these lines about setting the default storage class because they are covered in the Mitigation section.

Reply:
removed